examples/player/simid_player.js

```diff
@@ -430,11 +452,14 @@ class SimidPlayer {
   
   /** The creative wants to play video. */
   onRequestPlay(incomingMessage) {
```

Comment:
You should probably reject if non linear

Reply:
Done!